ano/environment.py

```diff
@@ -91,7 +91,7 @@ class Environment(dict):
     elif platformSystem == 'Windows':
         arduino_user_dir_guesses.insert(0, os.path.expanduser(os.path.join("~", "My Documents", "Arduino")))
 
-    default_board_model = 'uno'
+    default_board_model = 'mega2560'
```

Comment:
Sorry, this is my mistake. Apparently `ano build -h` reads from ano.ini outputs the board specified there as the default. So if I execute that command in my project root, it says mega2560 is default. If I execute it elsewhere that doesn't have a project (or ano.ini), like my home directory, it changes back to the uno as the default.

Do you think that behavior is appropriate? It seems like output from a help command should stay constant. If you agree, go ahead and open an issue for it and I can try and take care of it. Otherwise just forget I said anything! Thanks!